helix-term/src/ui/completion.rs

```diff
@@ -15,8 +15,16 @@ use helix_view::{
 use crate::commands;
 use crate::ui::{menu, Markdown, Menu, Popup, PromptEvent};
 
-use helix_lsp::{lsp, util};
-use lsp::CompletionItem;
+use helix_lsp::{lsp, util, OffsetEncoding};
+
+#[derive(Clone)]
+pub enum CompletionItem {
```

Comment:
Should this just be a struct?

Reply:
Yeah, isolated in the context of this PR this should be a struct, I used an enum, because I've extended this directly afterwards in https://github.com/Philipp-M/helix/blob/1dc2381ac6340642680d6063c362023ff212c1fe/helix-term/src/ui/completion.rs#L43 as I think `CompletionItem`s will likely be independent on LSP messages in the future (as seen in the linked PR).

But I can convert it into a struct for this PR, as I see more potential discussion related to this idea (unrelated to this PR).

Reply:
Oh, I see, that makes sense. Personally I don't see a need to change this to a struct only to change it back to an enum in a later PR. But I'll let @archseer make the call here.